authorization-common/src/main/java/com/sequenceiq/authorization/service/PermissionCheckService.java

```diff
@@ -91,6 +92,16 @@ public Object hasPermission(ProceedingJoinPoint proceedingJoinPoint) {
         return commonPermissionCheckingUtils.proceed(proceedingJoinPoint, methodSignature, startTime);
     }
 
+    private Optional<String> getInitiatorUserCrn(ProceedingJoinPoint proceedingJoinPoint, MethodSignature methodSignature) {
```

Comment:
The method name could be confusing, because it persists the initiator user as well.

Reply:
right, i'll rename it to make it easier to understand